packages/lesswrong/lib/collections/users/custom_fields.ts

```diff
@@ -1436,6 +1436,20 @@ addFieldsDict(Users, {
     canRead: ['guests'],
     ...schemaDefaultValue(0)
   },
+
+  tagRevisionCount: {
```

Comment:
I ran this and the counts seem correct now

`scripts/serverShellCommand.sh "Globals.recomputeDenormalizedValues({collectionName: 'Users', fieldName: 'tagRevisionCount'})"`